pcsx2/DEV9/ATA/ATA.h

```diff
@@ -0,0 +1,269 @@
+	bool ioRunning = false;
+	std::mutex ioMutex;
+
+	std::condition_variable ioThreadIdle_cv;
+	bool ioThreadIdle_bool = false;
+
+	std::condition_variable ioReady;
+	std::atomic_bool ioClose;
+	bool ioWrite;
+	bool ioRead;
+	void (ATA::*waitingCmd)() = nullptr;
+	//Write Buffer(s)
+
+	//Read Buffer
+	int rdTransferred = 0;
+	int wrTransferred = 0;
+	//Max tranfer on 24bit is 256*512 = 128KB
+	//Max tranfer on 48bit is 65536*512 = 32MB
+	int readBufferLen;
+	u8* readBuffer = nullptr;
+	//Read Buffer
+
+	//PIO Buffer
+	int pioPtr;
+	int pioEnd;
+	u8 pioBuffer[512];
+	
+	int sectorsPerInterrupt;
+	void (ATA::*pioDRQEndTransferFunc)() = nullptr;
+	//PIO Buffer
+
+	//Smart
+	bool smartAutosave = true;
+	bool smartErrors = false;
+	u8 smartSelfTestCount = 0;
+	//Smart
+
+	u8 sceSec[256 * 2] = {0};
+
+public:
+	ATA();
+
+	int Open(std::string hddPath);
+	void Close();
+
+	void ATA_HardReset();
+
+	u16 Read16(u32 addr);
+	void Write16(u32 addr, u16 value);
+
+	void Async(u32 cycles);
+
+	void ATAreadDMA8Mem(u8* pMem, int size);
+	void ATAwriteDMA8Mem(u8* pMem, int size);
+
+	u16 ATAreadPIO();
+	//ATAwritePIO;
+
+ private:
+	//Info
+	void CreateHDDinfo(int sizeMb);
+	void CreateHDDinfoCsum();
+
+	//State
+	void ResetBegin();
+	void ResetEnd(bool hard);
+
+	u8 GetSelectedDevice()
+	{
+		return (regSelect >> 4) & 1;
+	}
+	void SetSelectedDevice(u8 value)
+	{
+		if (value == 1)
+			regSelect |= (1 << 4);
+		else
+			regSelect &= ~(1 << 4);
+	}
+
+	u64 HDD_GetLBA();
+	void HDD_SetLBA(u64 sectorNum);
+
+	bool HDD_CanSeek();
+	bool HDD_CanAccess(int* sectors);
+
+	void ClearHOB();
+
+	//Transfer
+	void IO_Thread();
+	void IO_Read();
+	bool IO_Write();
+	void HDD_ReadAsync(void (ATA::*drqCMD)());
+	void HDD_ReadSync(void (ATA::*drqCMD)());
+	bool HDD_CanAssessOrSetError();
+	void HDD_SetErrorAtTransferEnd();
+
+	void QueueWrite(u64 sector, u8* data, u32 length);
+	bool DequeueWrite(u64* sector, u8** data, u32* length);
+	bool IsQueueEmpty();
+
+	//Commands
+	void IDE_ExecCmd(u16 value);
+
+	bool PreCmd();
+	void HDD_Unk();
+
+	void IDE_CmdLBA48Transform(bool islba48);
+
+	void DRQCmdDMADataToHost();
+	void PostCmdDMADataToHost();
+	void DRQCmdDMADataFromHost();
+	void PostCmdDMADataFromHost();
+	void HDD_ReadDMA(bool isLBA48);
+	void HDD_WriteDMA(bool isLBA48);
+	
+	void PreCmdExecuteDeviceDiag();
+	void PostCmdExecuteDeviceDiag();
+	void HDD_ExecuteDeviceDiag();
+
+	void PostCmdNoData();
+	void CmdNoDataAbort();
+	void HDD_FlushCache();
+	void HDD_InitDevParameters();
+	void HDD_ReadVerifySectors(bool isLBA48);
+	void HDD_SeekCmd();
+	void HDD_SetFeatures();
+	void HDD_SetMultipleMode();
+	void HDD_Nop();
+	void HDD_Idle();
+
+	void DRQCmdPIODataToHost(u8* buf, int buffLen, int buffIndex, int size, bool sendIRQ);
+	void PostCmdPIODataToHost();
+	void HDD_IdentifyDevice();
+
+	void HDD_ReadMultiple(bool isLBA48);
+	void HDD_ReadSectors(bool isLBA48);
+	void HDD_ReadPIO(bool isLBA48);
+	void HDD_ReadPIOS2();
+	void HDD_ReadPIOEndBlock();
+	//HDd_Write*
```

Comment:
Woops

Reply:
Yeah there is Hdd inconsistenties, but thats just some nitpicks

Reply:
I think HddCreate is the main inconsistency (which I can easily fix if desired)

I also have variables such as hddImage, but that is lowercase because of variable naming, not sure what else

Reply:
Well if it's distinct and somewhat consistent, i'm fine with it. I don't want to damage actual function over some formatting but it was more for readability in the first place.